xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -828,8 +828,15 @@ void CRenderManager::FlipPage(volatile std::atomic_bool& bStop, double pts /* =
         presentmethod = PRESENT_METHOD_BOB;
         invert = true;
       }
+      else if (interlacemethod == VS_INTERLACEMETHOD_IMX_ADVMOTION_DOUBLE)
```

Comment:
@FernetMenta 

would it be possible for instance extend AutoInterlaceMethod() with optional parameter - mInt could be passed and platform dependent methods could be mapped to one of that general type?